responses.go

```diff
@@ -44,30 +44,43 @@ func WriteChannelEventSuccess(w http.ResponseWriter, r *http.Request, event Chan
 }
 
 // WriteMsgSuccess writes a JSON response for the passed in msg indicating we handled it
-func WriteMsgSuccess(w http.ResponseWriter, r *http.Request, msg Msg) error {
-	LogMsgReceived(r, msg)
-	return writeData(w, http.StatusOK, "Message Accepted",
-		&msgReceiveData{
-			msg.Channel().UUID(),
-			msg.UUID(),
-			msg.Text(),
-			msg.URN(),
-			msg.Attachments(),
-			msg.ExternalID(),
-			msg.ReceivedOn(),
-		})
+func WriteMsgSuccess(w http.ResponseWriter, r *http.Request, msgs []Msg) error {
+
+	data := []msgReceiveData{}
+	for _, msg := range msgs {
+		LogMsgReceived(r, msg)
+		data = append(
+			data,
+			msgReceiveData{
+				msg.Channel().UUID(),
+				msg.UUID(),
+				msg.Text(),
+				msg.URN(),
+				msg.Attachments(),
+				msg.ExternalID(),
+				msg.ReceivedOn(),
+			})
+	}
+	return writeData(w, http.StatusOK, "Message Accepted", data)
```

Comment:
Feels slightly weird to me that now data can be a JSON list or a JSON dictionary. I feel like it might be friendlier to callers if we made sure data was always a dictionary. So in this case we would want to wrap the `msgReceiveData` in another struct where that was a key like `msgs`.